cmd/helm/install.go

```diff
@@ -462,3 +464,25 @@ func checkDependencies(ch *chart.Chart, reqs *chartutil.Requirements) error {
 	}
 	return nil
 }
+
+//readFile load a file from the local directory or a remote file with a url.
+func readFile(filePath string) ([]byte, error) {
+	_, err := os.Stat(filePath)
+	if err != nil {
+
+		resp, err := http.Get(filePath)
+		if err != nil {
+			return nil, err
```

Comment:
I have just learned GO for nearly  a month. There is too much I do not know. Your suggestion is so useful. Thank you for your help!